src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -152,9 +152,9 @@ public override async Task<string> GetStampName(string subscriptionId, string re
             JToken obj2 = ((JArray)siteObjects)
                     .Select(i => (JObject)i)
                     .FirstOrDefault(
-                        j => (j.ContainsKey("subscription") && j["subscription"]["name"].ToString() == subscriptionId
-                            && j.ContainsKey("resource_group_name") && j["resource_group_name"].ToString() == resourceGroupName
-                            && j.ContainsKey("stamp")));
+                        j => j.ContainsKey("subscription") && j["subscription"]["name"].ToString().ToLower() == subscriptionId.ToLower()
```

Comment:
nit: string.Equal("", "", StringComparison.OrdinalIgnoreCase)

Reply:
Oh right, forgot C# has a helper for everything